tests/views/test_template_view.py

```diff
@@ -8,24 +8,34 @@ class TestTemplateView(TestClient):
 
     def _upload_template(self, template_id, data):
         response = self.client.post('/template/{}'.format(template_id), content_type='application/json',
-                                    data=json.dumps(data))
+                                    data=json.dumps(data), headers=self.get_auth_headers())
         self.assertStatus(response, 201)
 
     def test_comms_template_upload(self):
         # When a valid comms template is uploaded
         data = dict(id="cb0711c3-0ac8-41d3-ae0e-567e5ea1ef99", label="test data", type="EMAIL", uri="test-uri.com",
                     classification={"GEOGRAPHY": "NI"})
         response = self.client.post('/template/cb0711c3-0ac8-41d3-ae0e-567e5ea1ef99', content_type='application/json',
-                                    data=json.dumps(data))
+                                    data=json.dumps(data), headers=self.get_auth_headers())
 
         # Then it is uploaded successfully with a 201 response
         self.assertStatus(response, 201)
 
-    def test_invalid_comms_template_upload_(self):
+    def test_create_comms_template_without_basic_auth(self):
+        # When a valid comms template is uploaded without correct basic auth
+        data = dict(id="cb0711c3-0ac8-41d3-ae0e-567e5ea1ef99", label="test data", type="EMAIL", uri="test-uri.com",
+                    classification={"GEOGRAPHY": "NI"})
+        response = self.client.post('/template/cb0711c3-0ac8-41d3-ae0e-567e5ea1ef99', content_type='application/json',
+                                    data=json.dumps(data))
+
+        # Then it isn't uploaded and we receive a 403 response
```

Comment:
`# Then it isn't uploaded and we receive a 403 response` -> `# Then it isn't uploaded and we receive a 401 response`

Reply:
@benjefferies not sure if you had any ideas on how to split up the test files as they're starting to get a bit long or is that alright?